cmd/admin-trace.go

```diff
@@ -232,11 +232,15 @@ func (s shortTraceMsg) String() string {
 		fmt.Fprintf(b, "?%s ", s.Query)
 	}
 	fmt.Fprintf(b, " %s ", s.Client)
-	fmt.Fprint(b, console.Colorize("HeaderValue", fmt.Sprintf("%2s ", s.CallStats.Duration.Round(time.Microsecond))))
-	fmt.Fprint(b, console.Colorize("Stat", fmt.Sprintf(" 🠉 ")))
-	fmt.Fprint(b, console.Colorize("HeaderValue", humanize.IBytes(uint64(s.CallStats.Rx))))
-	fmt.Fprint(b, console.Colorize("Stat", fmt.Sprintf("  🠋 ")))
-	fmt.Fprint(b, console.Colorize("HeaderValue", humanize.IBytes(uint64(s.CallStats.Tx))))
+	spaces := 15 - len(s.Client)
+	fmt.Fprintf(b, "%*s", spaces, " ")
+	fmt.Fprintf(b, console.Colorize("HeaderValue", fmt.Sprintf("  %2s", s.CallStats.Duration.Round(time.Microsecond))))
+	spaces = 12 - len(fmt.Sprintf("%2s", s.CallStats.Duration.Round(time.Microsecond)))
```

Comment:
In my experiment [here](https://play.golang.org/p/NOqEeDeJY0L), `spaces` is negative.

Reply:
@wlan0 did you check this comment ^^ ?

Reply:
This issue cannot happen here because of `CallStats.Duration`, which is filled in from `madmin.TraceInfo`

```
cmd/http-tracer.go:225:        t.CallStats = trace.CallStats{Latency: rs.Time.Sub(rq.Time), InputBytes: reqBodyRecorder.Size(),
```

As you can see, the latency is measured as the difference between two times. It prints as a value like `14.265105s` - examples shown below

```
$GOPATH/src/github.com/minio/mc$ mc admin trace play
22:42:59.580 [200 OK] s3.PutObject play.min.io/mc-test-bucket-22145/datafile-65-MB 147.75.201.93     14.265105s   🠉 65 MiB  🠋 331 B
22:43:14.281 [200 OK] s3.GetBucketLocation play.min.io/mc-test-bucket-22145/?location=  147.75.201.93     706µs       🠉 132 B  🠋 422 B
22:43:14.373 [200 OK] s3.HeadBucket play.min.io/mc-test-bucket-22145/ 147.75.201.93     419µs       🠉 132 B  🠋 253 B
22:43:14.478 [200 OK] s3.ListObjectsV1 play.min.io/mc-test-bucket-22145/?delimiter=%2F&max-keys=1000&prefix=  147.75.201.93     3.484ms      🠉 132 B  🠋 1.5 KiB
22:43:15.124 [200 OK] s3.GetBucketLocation play.min.io/mc-test-bucket-22145/?location=  147.75.201.93     642µs       🠉 132 B  🠋 422 B
22:43:15.218 [200 OK] s3.HeadBucket play.min.io/mc-test-bucket-22145/ 147.75.201.93     452µs       🠉 132 B  🠋 253 B
22:43:15.282 [200 OK] s3.HeadBucket play.min.io/mc-test-bucket-22145/ 147.75.201.93     417µs       🠉 132 B  🠋 253 B
22:43:15.348 [200 OK] s3.ListObjectsV1 play.min.io/mc-test-bucket-22145/?delimiter=&max-keys=1000&prefix=  147.75.201.93     3.238ms      🠉 132 B  🠋 1.5 KiB
22:43:15.415 [200 OK] s3.HeadBucket play.min.io/mc-test-bucket-22145/ 147.75.201.93     418µs       🠉 132 B  🠋 253 B
22:43:15.477 [200 OK] s3.ListObjectsV1 play.min.io/mc-test-bucket-22145/?delimiter=%2F&max-keys=1000&prefix=  147.75.201.93     3.858ms      🠉 132 B  🠋 1.5 KiB
22:43:15.548 [200 OK] s3.DeleteMultipleObjects play.min.io/mc-test-bucket-22145/?delete=  147.75.201.93     5.805ms      🠉 303 B  🠋 401 B
22:43:15.615 [204 No Content] s3.DeleteBucket play.min.io/mc-test-bucket-22145/ 147.75.201.93     2.165ms      🠉 132 B  🠋 261 B
22:43:15.680 [404 Not Found] s3.GetBucketLocation play.min.io/mc-test-bucket-22145/?location=  147.75.201.93     702µs       🠉 132 B  🠋 631 B

```

In contrast, in the play.golang.org file, the string value of `fmt.Sprintf("%2s", t.Round(time.Microsecond))` is `-0001-11-30 12:15:30.918274 +0000 UTC`  - a timestamp.

Reply:
@wlan0 oops, I didn't notice that `time.Time` also had a `Round` and hence the confusion.